R/geom-rug.r

```diff
@@ -66,6 +66,10 @@ GeomRug <- ggproto("GeomRug", Geom,
     rugs <- list()
     data <- coord$transform(data, panel_params)
 
+    if (inherits(coord, 'CoordFlip')) {
+      sides <- chartr('tblr', 'rlbt', sides)
```

Comment:
Would you mind adding a comment to this line explaining what's happening?

Reply:
I have added a comment to the block.